src/main/java/ca/nines/ise/writer/XMLWriter.java

```diff
@@ -16,179 +16,1303 @@
-          }
-
-          while (splitStack.size() > 0) {
-            Element popped = splitStack.pop();
-            Element clone = xml.createElement(popped.getNodeName());
-            NamedNodeMap attrs = popped.getAttributes();
-            int numAttrs = attrs.getLength();
-            for (int i = 0; i < numAttrs; i++) {
-              clone.setAttribute(attrs.item(i).getNodeName(), attrs.item(i).getNodeValue());
-            }
-            xmlStack.peekFirst().appendChild(clone);
-            xmlStack.push(clone);
-          }
-
-          break;
-        case START:
-          StartNode startNode = (StartNode) n;
-          Element startElement = xml.createElement(startNode.getName().toLowerCase());
-          for (String name : startNode.getAttributeNames()) {
-            startElement.setAttribute(name, startNode.getAttribute(name));
-          }
-          xmlStack.peekFirst().appendChild(startElement);
-          xmlStack.push(startElement);
-          break;
-        case TEXT:
-          Text t = xml.createTextNode(n.getText());
-          xmlStack.peekFirst().appendChild(t);
-          break;
-        default:
-          throw new Exception("Cannot convert " + n.getName() + " to XML");
-      }
-    }
-
-    try {
-      Transformer transformer = TransformerFactory.newInstance().newTransformer();
-      transformer.setOutputProperty(OutputKeys.INDENT, "no");
-      transformer.setOutputProperty(OutputKeys.OMIT_XML_DECLARATION, "yes");
-      transformer.setOutputProperty(OutputKeys.ENCODING, "UTF-8");
-      DOMSource source = new DOMSource(xml);
-      StreamResult stream = new StreamResult(out);
-      transformer.transform(source, stream);
-    } catch (Exception ex) {
-      ex.printStackTrace();
-    }
-  }
-
-  /**
-   * Unsupported.
-   *
-   * @param dom
-   * @param ann
-   * @throws TransformerConfigurationException
-   * @throws TransformerException
-   * @throws IOException
-   * @throws Exception
-   */
-  @Override
-  public void render(DOM dom, Annotation ann) throws TransformerConfigurationException, TransformerException, IOException, Exception {
-    throw new UnsupportedOperationException("Not supported."); //To change body of generated methods, choose Tools | Templates.
-  }
+
+import nu.xom.Attribute;
+import nu.xom.Document;
+import nu.xom.Element;
+import nu.xom.Elements;
+import nu.xom.Text;
+import nu.xom.Serializer;
+
+public class XMLWriter extends Writer{
+	public static final String DOC_NS = "http://internetshakespeare.uvic.ca/exist/rest/db/apps/iseapp/content/schema/text/documentClass/dramaticWork.rng";
+    public static final String LINK_NS = "http://internetshakespeare.uvic.ca/#internal-linking";
+    public static final String HTML_NS = "http://www.w3.org/1999/xhtml";
+    
+    static final Map<String , String> XML_MAP = new HashMap<String , String>() {{
+    	put("RA", "right");
+    	put("C", "center");
+    	put("J", "justify");
+    	put("TITLEHEAD","title");
+    	put("VERSEQUOTE","quote");
```

Comment:
TITLEHEAD and VERSEQUOTE shouldn't be handled in the serializer at all -- they're deprecated, and should be either handled by a deprecation transformer (see #10 ) before the serialization step and/or treated as unrecognized elements.